packages/web/src/app/contexts/api/moonpay/provider.ts

```diff
@@ -12,8 +12,10 @@ import type {
 } from './types';
 import { buildParams } from './utils';
 
+const ERROR_STATUS_CODE = 400;
+
 const baseParams: MoonpayBaseParams = {
-  apiKey: MOONPAY_API_KEY!,
+  apiKey: String(MOONPAY_API_KEY),
```

Comment:
I don't think that it should be changed

Reply:
There is rule `@typescript-eslint/no-non-null-assertion` that do not allow such syntax